native_client/dotnet/DeepSpeechConsole/DeepSpeechConsole.csproj

```diff
@@ -62,7 +62,8 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="arctic_a0024.wav">
+    <Content Include="..\..\..\data\smoke_test\arctic_a0024.wav">
+      <Link>arctic_a0024.wav</Link>
```

Comment:
There's no good reason to keep that file, is there @carlfm01 ?

Reply:
@carlfm01 if it was only for some local test I can remove it and use one of the existing files from `data\smoke_test`

Reply:
Yes, we can use the audio from data